src/org/usfirst/frc/team166/robot/OI.java

```diff
@@ -67,10 +67,13 @@ public OI() {
 
 		buttonA.whenPressed(new DriveDistance(31.4, 0.5));
 
-		if (!isReversed)
+		if (!isReversed) {
```

Comment:
This initialization still only gets run once - it won't toggle like you're expecting